packages/app/src/components/inspector/TextStyleInspector.tsx

```diff
@@ -0,0 +1,377 @@
+        case paragraphInputId:
+          return <Label.Label>Paragraph</Label.Label>;
+        default:
+          return null;
+      }
+    },
+    [characterInputId, lineInputId, paragraphInputId],
+  );
+
+  return (
+    <InspectorPrimitives.Section>
+      <InspectorPrimitives.SectionHeader>
+        <InspectorPrimitives.Title>Text</InspectorPrimitives.Title>
+      </InspectorPrimitives.SectionHeader>
+
+      <Spacer.Vertical size={10} />
+      <InspectorPrimitives.Row>
+        <Select
+          id="font-family"
+          value={fontFamily}
+          options={fontFamilies}
+          getTitle={(name) => name}
+          onChange={onChangeFontFamily}
+        />
+      </InspectorPrimitives.Row>
+
+      <Spacer.Vertical size={6} />
+      <InspectorPrimitives.Row>
+        <Select
+          id="font-weight"
+          value={fontWeight}
+          options={fontSize}
+          getTitle={(name) => name}
+          onChange={onChangeFontWeight}
+        />
+        <Spacer.Horizontal size={8} />
+        <InputField.Root id="font-size" size={50}>
+          <InputField.NumberInput
+            value={32}
+            onSubmit={() => {}}
+            onNudge={() => {}}
+          />
+          <InputField.Label>px</InputField.Label>
+        </InputField.Root>
+      </InspectorPrimitives.Row>
+
+      <Spacer.Vertical size={6} />
+      <InspectorPrimitives.Row id={'text-aligment'}>
+        <LabeledElementView renderLabel={renderLabel}>
+          <InputField.NumberInput
+            id={characterInputId}
+            value={32}
+            onSubmit={() => {}}
+            onNudge={() => {}}
+          />
+          <Spacer.Horizontal size={8} />
+          <InputField.NumberInput
+            id={lineInputId}
+            value={32}
+            onSubmit={() => {}}
+            onNudge={() => {}}
+          />
+          <Spacer.Horizontal size={8} />
+          <InputField.NumberInput
+            id={paragraphInputId}
+            value={32}
+            onSubmit={() => {}}
+            onNudge={() => {}}
+          />
+          <Spacer.Horizontal size={8} />
+          <ColorInputFieldWithPicker
+            id={'colorInputId'}
+            value={fontColor}
+            onChange={onChangeFontColor}
+          />
+        </LabeledElementView>
+      </InspectorPrimitives.Row>
+    </InspectorPrimitives.Section>
+  );
+});
+const TextAligmentRow = memo(function TextAligmentRow({
+  fontAlignment,
+  fontHorizontalAlignment,
+  fontVerticalAlignment,
+  onChangeFontAlingment,
+  onChangeFontHorizontalAlignment,
+  onChangeFontVerticalAlignment,
+}: TextAligmentRowProps) {
+  return (
+    <InspectorPrimitives.Section>
+      <InspectorPrimitives.SectionHeader>
+        <InspectorPrimitives.Title>Alignment</InspectorPrimitives.Title>
+      </InspectorPrimitives.SectionHeader>
+      <Spacer.Vertical size={10} />
+      <InspectorPrimitives.Row>
+        <LabeledElementView renderLabel={() => 'Auto Height'}>
```

Comment:
I think if we make it a `Label.Label` it should be the right size?

```tsx
renderLabel={() => <Label.Label>Auto Height</Label.Label>}
```

Reply:
oh! make sense 😅